scripts/scripts.js

```diff
@@ -45,15 +34,41 @@ async function loadFonts() {
  * Builds all synthetic blocks in a container element.
  * @param {Element} main The container element
  */
-function buildAutoBlocks(main) {
+function buildAutoBlocks(/* main */) {
   try {
-    buildHeroBlock(main);
+    // buildHeroBlock(main);
   } catch (error) {
     // eslint-disable-next-line no-console
     console.error('Auto Blocking failed', error);
   }
 }
 
+function decorateTIAAContent(main) {
+  main.querySelectorAll('h1').forEach((h1) => {
+    h1.className = 'display-strong-1';
+  });
+  main.querySelectorAll('h2').forEach((h2) => {
+    h2.className = 'eyebrow-2 color-accent-tertiary';
+  });
+  main.querySelectorAll('h3').forEach((h3) => {
+    h3.className = 'display-3';
+  });
+  main.querySelectorAll('h1>u,h2>u,h3>u').forEach((h) => {
```

Comment:
Is the `u` tag generated from the authoring? I'm curious in what cases we need it used.